docs/.vitepress/theme/custom.css

```diff
@@ -129,6 +159,225 @@ html.dark body {
+  position: relative;
+}
+.VPHero::before {
+  content: '';
+  position: absolute;
+  bottom: 80px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 104px;
+  background: url('stars-background.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.VPHero::after {
+  content: '';
+  position: absolute;
+  bottom: -8px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 8px;
+  background: url('hero-wavy-line.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.VPHero .container .main .name .clip {
+  --vp-home-hero-name-color: #F6D553;
+  font-weight: 700;
+  font-size: 52px;
+  text-transform: uppercase;
+}
+.VPHero .container .main .tagline {
+  --vp-c-text-2: #FFFBD5;
+  max-width: 24ch;
+}
+.VPHero .container .image .image-container .VPImage {
+  max-width: 578px;
+  max-height: 238px;
+  width: 100%;
+}
+@media (min-width: 768px) {
+  .VPHero .container .image .image-container .VPImage {
+    max-width: 578px;
+    max-height: 238px;
+    width: unset;
+  }
+}
+.VPHomeFeatures .item .VPLink {
+  --vp-c-bg-soft: #131B2D;
+  max-width: 557px;
+  max-height: 351px;
+  border-radius: 8px;
+}
+.VPHomeFeatures .item .VPLink .box {
+  display: grid;
+  grid-template-columns: 0.9fr 1.1fr;
+  grid-template-areas:
+    "title icon"
+    "details icon"
+    "link icon"
+    "link-text icon";
+  padding: 60px 38px 60px 38px;
+  }
+.VPHomeFeatures .item .VPLink .box .VPImage {
+  grid-area: icon;
+  justify-self: end;
+}
+.VPHomeFeatures .item .VPLink .box .title {
+  grid-area: title;
+  color: #F6D553;
+  font-size: 36px;
+  line-height: 52px;
+}
+.VPHomeFeatures .item .VPLink .box .details {
+--vp-c-text-2: #FFFBD5;
+  grid-area: details;
+  font-size: 22px;
+  line-height: 32px;
+}
+.VPHomeFeatures .item .VPLink .link-text {
+  grid-area: link;
+  color: #FFFBD5;
+}
+.VPHomeFeatures .item .VPLink .link-text-value {
+  --vp-c-text-2: #FFFBD5;
+  grid-area: link-text;
+  color: var(--vp-c-text-2);
+  font-size: 16px;
+  text-decoration: underline;
+}
+.VPButton {
+  --vp-button-brand-text: #FFFBD5;
+  color: var(--vp-button-brand-text);
+}
+.VPNavBarTitle .title {
+  border-bottom: none;
+}
+.VPNavBarTitle .title .VPImage {
+  height: 96px;
+}
+
+.VPNavBarTitle .title span {
+  display: none;
+}
+
+.VPContent .VPDoc .container .aside .aside-curtain {
+  display: none;
+}
+
+.VPNavBar .wrapper .container .content {
+  background-color: #131B2D;
+}
+
+.VPLink.VPNavBarMenuLink {
+  --vp_c-text-1: #FFFBD5;
+  color: var(--vp_c-text-1);
+  font-weight: 400;
+}
+.VPLink.VPNavBarMenuLink:hover {
+  --vp_c-text-1: #F6D553;
+  color: var(--vp_c-text-1);
+  
+}
+.VPLink.VPNavBarMenuLink:active {
+  --vp_c-text-1: #F6D553;
+  color: var(--vp_c-text-1);
+  text-decoration: underline;
+}
+.VPNavBarSearch #local-search .DocSearch {
+  background-color: transparent;
+}
+
+.DocSearch .DocSearch-Button-Keys {
+  display: none;
+}
+
+
+.DocSearch-Button-Container .DocSearch-Button-Placeholder {
+  display: none;
+}
```

Comment:
We may not show those elements, but they are still in the DOM (see code snippet below). I wonder if we have a way to customise that search container using a different approach. I mean, this particular example might be ok - I don't think screen reader users would be affected, because they should hear button's aria-label announcement only. So I'm more wondering than requesting any change here :)
```
<div id="local-search">
  <button type="button" class="DocSearch DocSearch-Button" aria-label="Search">
    <span class="DocSearch-Button-Container">
      <span class="vp-icon DocSearch-Search-Icon"></span>
      <span class="DocSearch-Button-Placeholder">Search</span>
      <span class="DocSearch-Button-Keys">
        <kbd class="DocSearch-Button-Key"></kbd>
        <kbd class="DocSearch-Button-Key">K</kbd>
      </span>
  </button>
</div>

Reply:
Oh you're right! Yeah I should look into another way so it is also removed from the DOM! 👍